RealmSwift/Projection.swift

```diff
@@ -321,15 +344,21 @@ extension ProjectionObservable {
 
             var projectedChanges = [PropertyChange]()
             for i in 0..<newValues.count {
-                for property in schema.filter({ $0.originPropertyKeyPathString == names[i] }) {
+                for property in schema.filter({ prop in
```

Comment:
Now that the filter block is multiple lines of code it should be moved to a separate statement, as putting it inline in the `for in` makes it hard to see where the filter ends and the body of the for begins.